DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -407,6 +412,11 @@ private function normalizeDriverOptions(array $connection)
             $driverOptions['context'] = new Reference($driverOptions['context']);
         }
 
+        $driverOptions['driver'] = [
+            'name' => 'symfony-mongodb',
+            'version' => self::getODMVersion(),
```

Comment:
would it have more value if augmented with bundle's version as well?

Reply:
I don't think so. I'm not sure what other hosters use this for, but at MongoDB we use the data from MongoDB Atlas to judge version adoption and to influence strategic decisions (e.g. around what PHP versions to support/not support). In that sense the relevant info for us is a) the fact that someone is using the ODM through Symfony, and b) what ODM version they are using.